openverse_catalog/dags/maintenance/rotate_db_snapshots.py

```diff
@@ -0,0 +1,91 @@
+"""
+Manages weekly database snapshots. RDS does not support weekly snapshots
+schedules on its own, so we need a DAG to manage this for us.
+
+It runs on Saturdays at 00:00 UTC in order to happen before the data refresh.
+
+The DAG will automatically delete the oldest snapshots when more snaphots
+exist than it is configured to retain.
+
+Requires two variables:
+
+`AIRFLOW_RDS_ARN`: The ARN of the RDS DB instance that needs snapshots.
+`AIRFLOW_RDS_SNAPSHOTS_TO_RETAIN`: How many historical snapshots to retain.
+"""
+
+import logging
+from datetime import datetime
+
+import boto3
+from airflow.decorators import dag, task
+from airflow.models import Variable
+from airflow.providers.amazon.aws.operators.rds import RdsCreateDbSnapshotOperator
+from airflow.providers.amazon.aws.sensors.rds import RdsSnapshotExistenceSensor
+
+
+logger = logging.getLogger(__name__)
+
+DAG_ID = "check_silenced_dags"
```

Comment:
This is causing duplication errors since this ID is already being used by the `check_silenced_dags` DAG: https://github.com/WordPress/openverse-catalog/blob/06335539a187036435e0cb02fa94ee4324a7c8da/openverse_catalog/dags/maintenance/check_silenced_dags.py#L35

Maybe:
```suggestion
DAG_ID = "rotate_db_snapshots"
```

Reply:
Oh dear, I copied this and _definitely_ meant to change the text but just totally forgot what I was doing last week :facepalm: The name is clearly not correct, thank you for the suggested change.